test_unstructured_ingest/expected-structured-output/azure/spring-weather.html.json

```diff
@@ -64,17 +64,16 @@
   },
   {
     "type": "Title",
-    "element_id": "568c824acda361cfc270a75e2eca7a23",
-    "text": "Weather.gov >",
+    "element_id": "6ef25d028f4c859b5d955c777c4c597e",
+    "text": "Weather.gov > News Around NOAA > Are You Weather-Ready for the Spring?",
```

Comment:
This looks like a good change to me, with link texts captured below

Reply:
Yeah, the HTML behind this looks like this:
```html
<div>
  <a href="https://www.weather.gov">Weather.gov</a>
  &gt; <a href="https://www.weather.gov/news">News Around NOAA</a>
  &gt; Are You Weather-Ready for the Spring?
</div>
```

All the content of the `<div>` is phrasing (inline) content so it's all considered the same paragraph and therefore Element.

The old parser would break the paragraph/element when there were two `<a>` elements in succession.

That's sometimes the right choice, but only when the `<a>` element is itself surrounding a block element like `<div>` or `<p>`.